keyboards/keebio/bdn9/keymaps/rishka/keymap.c

```diff
@@ -0,0 +1,67 @@
+/* Copyright 2019 Danny Nguyen <danny@keeb.io>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    /*
+        | Knob 1: Vol Dn/Up |      | Knob 2: Page Dn/Up |
+        | Press: Mute       | Home | Press: Play/Pause  |
+        | Hold: Layer 2     | Up   | RGB Mode           |
+        | Left              | Down | Right              |
+     */
+    [0] = LAYOUT(
+        KC_MUTE, KC_UP, TO(0),
+        KC_F12  , _______  , KC_F1,
+        KC_LEFT, KC_DOWN, KC_RGHT
+    ),
+    /*
+        | RESET          | N/A  | Media Stop |
+        | Held: Layer 2  | Home | RGB Mode   |
+        | Media Previous | End  | Media Next |
+     */
+    [1] = LAYOUT(
+        RESET  , KC_HOME, _______,
+        _______, _______, _______,
+        KC_MPRV, KC_END , KC_MNXT
+    ),
+};
+int current_layer = 0;
```

Comment:
You can use `layer_state` instead of this.  This is the layer mask, same as what `layer_state_set_user` uses.

Reply:
updated